### PR TITLE
fix: Add Ignore location for hass topics

### PR DESCRIPTION
**Pull request by billiaz**
Hass topics, are not respacting the ignoreLoc setting in config. Add support for this.

### CODE REVIEW

**Review by robertsLando — APPROVED** [review]
Thanks @billiaz !

File: lib/Gateway.js [hunk 1]
```diff
@@ -526,9 +526,9 @@ function rgbToHex (rgb) {
  * @param {any} node The Zwave Node Object
  * @returns A string in the format [<location>-]<name>, if location doesn't exist it will be ignored, if the node name doesn't exists the node id with node prefix string will be used
  */
-function getNodeName (node) {
+function getNodeName (node, ignoreLoc) {
   return (
-    (node.loc ? node.loc + '-' : '') +
+    (ignoreLoc ? (node.loc ? node.loc + '-' : '') : '') +
```
Comment (robertsLando) [inline lib/Gateway.js:531]:
```suggestion
    (!ignoreLoc && node.loc ? node.loc + '-' : '') +
```

**Review by robertsLando — CHANGES_REQUESTED** [review]

File: lib/Gateway.js (hunk 1 above)
Comment (robertsLando) [reply lib/Gateway.js:531]: @billiaz the old check does the oppsite of what you would like to do

**Comment by billiaz** [comment]
late night mistakes :) thanks

**Review by robertsLando — APPROVED** [review]
LGTM

**Pull request merged**